pom.xml

```
@@ -9,11 +9,11 @@
     </parent>
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>thycotic-devops-secrets-vault</artifactId>
-    <version>1.0.1</version>
+    <version>1.1.1</version>
```

nit: let's reset the last digit to 0
```suggestion
    <version>1.1.0</version>
```